pyvista/core/dataobject.py

```diff
@@ -628,7 +629,7 @@ def _config_user_dict(self: Self) -> None:
                 # When loaded from file, field will be cast as pyvista ndarray
                 # Convert to string and initialize new user dict object from it
                 self._user_dict = _SerializedDictArray(''.join(array))
-            elif isinstance(array, str) and repr(self._user_dict) != array:
+            elif isinstance(array, str) and repr(self._user_dict) != array:  # type: ignore[unreachable]
```

Comment:
This is fine to ignore here. I suspect some of these issues come from the `convert_array` overloads not allow a `str` return type (which is only used for things like `user_dict`